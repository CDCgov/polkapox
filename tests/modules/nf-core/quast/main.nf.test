nextflow_process {

    name "Test Process QUAST"
    script "modules/nf-core/quast/main.nf"
    process "QUAST"

    test("Quast Test") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                input[0] = Channel.of(
                    file("${projectDir}/tests/nf-test-data/test_sub.contigs.fasta", checkIfExists: true)
                )
                input[1] = true
                input[2] = true
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
