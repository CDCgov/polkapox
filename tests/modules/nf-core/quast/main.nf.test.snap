{
    "Quast Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "2": [
                    
                ],
                "results": [
                    
                ],
                "tsv": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-07T16:09:55.54097222"
    }
}